test/Microsoft.DotNet.Web.Authentication.Tests/Controllers/AnonymousAttributeController.cs

```diff
@@ -7,9 +7,9 @@
 namespace Microsoft.DotNet.Web.Authentication.Tests.Controllers;
 
 [Route("test-auth/anonymous")]
-[AllowAnonymous]
 public class AnonymousAttributeController : ControllerBase
 {
+    [AllowAnonymous]
```

Comment:
This one should just be deleted, since it's identical to the next test. Ideally, you'd have wanted another test that had the "inherit from the controller" behavior tested (you'd just need to not have any Authorize attributes in that controller to keep CodeQL quiet)